test/jdk/java/lang/invoke/RevealDirectTest.java

```diff
@@ -28,9 +28,6 @@
  * @compile -XDignore.symbol.file RevealDirectTest.java
  * @run junit/othervm -ea -esa test.java.lang.invoke.RevealDirectTest
  *
- * @test
- * @summary verify Lookup.revealDirect on a variety of input handles, with security manager
- * @run main/othervm/policy=jtreg.security.policy/secure=java.lang.SecurityManager -ea -esa test.java.lang.invoke.RevealDirectTest
```

Comment:
line 36 can also be removed.

```
* $ $JAVA8X_HOME/bin/java -cp $JUNIT4_JAR:../../../.. -ea -esa -Djava.security.manager test.java.lang.invoke.RevealDirectTest
```

Reply:
hasSM and the code that only runs when true can be deleted too.

Reply:
Fixed in https://github.com/openjdk/jdk/pull/21498/commits/559934662119b1372fd831de8be7c97f877e0947